mozilla_vpn/views/channel_group_proportions_table.view.lkml

```diff
@@ -19,6 +19,16 @@ view: +channel_group_proportions_table {
     sql: ${TABLE}.promotion_codes[SAFE_ORDINAL(1)] ;;
   }
 
+ dimension: plan_group {
+  description: "Indicates the plan type (1 year, 6 month, 1 month, 1 year_bundle)"
+  type: string
+  sql: CONCAT(IF(${product_name} LIKE "%Relay%", CONCAT("bundle", "_"), ""), ${plan_interval_count}, "_", ${plan_interval});;
```

Comment:
The description indicates a `_bundle` suffix, but the SQL adds a `bundle_` prefix.  One or the other should be changed so they agree.

Reply:
Very good catch! I updated the description :)